src/clr-angular/utils/drag-and-drop/index.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Type } from '@angular/core';
+
+import { ClrDragHandle } from './drag-handle';
+import { ClrDraggableGhost } from './draggable-ghost';
+import { ClrDraggable } from './draggable/draggable';
+import { ClrDroppable } from './droppable/droppable';
+import { ClrIfDragged } from './if-dragged';
+
+export * from './draggable/draggable';
+export * from './droppable/droppable';
+export * from './if-dragged';
+export * from './drag-handle';
+export * from './draggable-ghost';
+export * from './drag-event';
+
+export const DRAG_AND_DROP_DIRECTIVES: Type<any>[] = [
```

Comment:
This should be moved to the drag module instead. We should only be exporting from an index file.